src/state/roles/selectors.js

```diff
@@ -1,10 +1,24 @@
 import { createSelector } from 'reselect';
+import { getPermissionsList } from 'state/permissions/selectors';
 
 export const getRoles = state => state.roles;
 export const getRolesIdList = state => state.roles.list;
 export const getRolesMap = state => state.roles.map;
+export const getSelectedRolePermissions = state => state.roles.selected.permissions;
```

Comment:
`selected` can be null/undefined, these selectors are unsafe. Please use `state.roles.selected && state.roles.selected.permissions`, or `get(state.roles, 'selected.permissions')` (`get` is a function from lodash)